src/views/WordBazaar.vue

```diff
@@ -103,6 +102,7 @@ export default {
       this.wordRegistryState = await Backend.getWordRegistry();
     },
   },
+  metaInfo: { title: 'WordBazaar' },
```

Comment:
Shouldn't this also be part of the locales file and used as i18n token?

Reply:
Indeeed